### ISSUE TITLE
[FEATURE REQUEST] Remove / Blank date option

### ISSUE THREAD

**Issue by Turtlecute33** [emavgl/oinkoin#186]
Hi sir, first off all thank you for your job. I really love your app but i would like to suggest one UX improvement option: a default blank date settings.

I really love oinkoin but i (and many other people that i know) do not update every day their expense report but do it weekly or monthly. Add all the dates every time makes 3x time longer insert an entry. Would be nice to have a setting for disabling date insert in the UI. 

Thanks ^^

**Comment by emavgl** [comment]
Hi, I am not sure that I got the problem. Couldn't you selecr a fictional day? In which month / date should it put these entries?

**Comment by Turtlecute33** [comment]
Yeah, as a user, I could use a fictional or random date, but in this case, my home would end up cluttered with expenses spread across random days. I just think it would be nice to have the possibility with a new settings to remove the date selector and work only with months, pre-selecting the current month. This makes it much easier to input expenses if you don't do it daily and simplifies the UI by reducing unnecessary information (especially if you're not interested in displaying it)
